app/src/main/java/com/nitramite/courier/DHLActiveTrackingStrategy.java

```diff
@@ -1,95 +1,60 @@
 
-    // Host name verifier
-    public boolean verify(String hostname, SSLSession session) {
-        return true;
-    }
+    // Logging
+    private static final String TAG = "DHLActiveTracking";
 
 
     @Override
     public ParcelObject execute(String parcelCode) {
         ParcelObject parcelObject = new ParcelObject(parcelCode);
         ArrayList<EventObject> eventObjects = new ArrayList<>();
-        TrustManager[] trustAllCerts = new TrustManager[]{
-                new X509TrustManager() {
-                    public java.security.cert.X509Certificate[] getAcceptedIssuers() {
-                        return null;
-                    }
-                    @SuppressLint("TrustAllX509TrustManager")
-                    public void checkClientTrusted(
-                            java.security.cert.X509Certificate[] certs, String authType) {
-                    }
-                    @SuppressLint("TrustAllX509TrustManager")
-                    public void checkServerTrusted(
-                            java.security.cert.X509Certificate[] certs, String authType) {
-                    }
-                }
-        };
         try {
-            SSLContext sc = SSLContext.getInstance("SSL");
-            sc.init(null, trustAllCerts, new java.security.SecureRandom());
-            HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
-            String url = "https://www.logistics.dhl/DatPublic/search.do?autoSearch=true&l=fi&directDownload=XML&statusHistory=true&search=consignmentId&a=" + parcelCode;
-            URL obj = new URL(url);
-            HttpsURLConnection con = (HttpsURLConnection) obj.openConnection();
-            con.setConnectTimeout(5000);    // Timeout for connecting
-            con.setReadTimeout(5000);       // Timeout for reading content
-            con.setSSLSocketFactory(sc.getSocketFactory());
-            con.setHostnameVerifier(this);
-            con.setRequestMethod("GET");
-            String USER_AGENT = "Mozilla/5.0";
-            con.setRequestProperty("User-Agent", USER_AGENT);
-            con.setRequestProperty("Host", "www.logistics.dhl");
-            con.setRequestProperty("Connection", "keep-alive");
-            con.setRequestProperty("Cache-Control", "no-cache");
-            con.setRequestProperty("Accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8");
-            BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
-            // Get input steam
-            String inputLine;
-            StringBuilder response = new StringBuilder();
-            while ((inputLine = in.readLine()) != null) {
-                response.append(inputLine).append("\n");
-            }
-            // Close connection
-            in.close();
+            String url = "https://www.dhl.com/DatPublic/search.do?autoSearch=true&l=fi&directDownload=XML&statusHistory=true&search=consignmentId&a=" + parcelCode;
```

Comment:
Constant here also?

Reply:
Should I make part "https://www.dhl.com/DatPublic/search.do?autoSearch=true&l=fi&directDownload=XML&statusHistory=true&search=consignmentId&a=" private static final string? and then later append url + parcelCode? is that what you mean?

Reply:
Didn't check so carefully but yep it'd more consistent.